flowey/flowey_lib_hvlite/src/init_hyperv_tests.rs

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+//! Hyper-V test pre-reqs
+
+use flowey::node::prelude::*;
+
+flowey_request! {
+    pub struct Request(pub WriteVar<SideEffect>);
+}
+
+new_flow_node!(struct Node);
+
+impl FlowNode for Node {
+    type Request = Request;
+
+    fn imports(ctx: &mut ImportCtx<'_>) {
+        ctx.import::<crate::cfg_openvmm_magicpath::Node>();
+        ctx.import::<flowey_lib_common::download_protoc::Node>();
+    }
+
+    fn emit(requests: Vec<Self::Request>, ctx: &mut NodeCtx<'_>) -> anyhow::Result<()> {
+        // Only X64 for now, these are set manually on ARM64 runners
```

Comment:
Seems like you should do this on ARM64, too, even if you happen to know the runners are already correctly configured.

Reply:
Should be harmless.